src/main/java/cope/cosmos/client/features/modules/movement/NoSlow.java

```diff
@@ -107,7 +107,7 @@ public void onPacketSend(PacketEvent.PacketSendEvent event) {
 
     @SubscribeEvent
     public void onInputUpdate(InputUpdateEvent event) {
-        if (this.isSlowed()) {
+        if (this.isSlowed() && !ncpStrict.getValue()) {
```

Comment:
yeah but the check you do here makes 0 sense. until this is removed, its not gonna be merged cause it completely removes the clientsided noslow effect, which is the entire point of this module.

Reply:
and why dont you always not wanna do it? it legit is what makes you not be slowed down by items...

Reply:
what im trying to do is make it so noslow works normally if ur strafing in the air, but add an option where if your on the ground just walking around with noslow it wont do shit and just slow you down normally. 

perhaps we might need that code for noslow to work on the ground or while strafing.

Reply:
this isnt the right option though???? the fuck???? do you even read the descriptions of the settings and the code?

Reply:
the digging packet is what allows u to semi-bypass it but if you move like 0.5 blocks u start getting rubberbanded right? so shoulnd't you have a way to choose if u wanna get rubberbanded if u walk 0.5 blocks while eating, or to go slower if your on the ground but not get rubberbanded?

Reply:
what are you even talking about? that packet has nothing to do with rubberbanding on strict servers? you still have to multiply the strafe and forward values by 5.0f.

this is because Minecraft's code multiplies those two values by 0.2f when a hand is active to slow you down, and we reverse that with 5.0f.

regardless, you always want this. your thing here makes no sense.

Reply:
and the sneak bypass only works while strafing, nobody has figured out a way to bypass it without it... again, im not merging this pr until you fix this because this will entirely fuck up NoSlow on servers that dont require you to have a stricter NoSlow config.

Reply:
im not 100% sure what you aren't understanding, you adding this check makes 0 sense.

Reply:
and i understand what youre trying to say, but your implementation is fucking horrible.

a better way to do that is to check if the player is on the ground, and then not to do that if the sneak bypass is enabled, then it wouldn't rubberband you.

but doing this with no onground check and no thought behind it will hinder the use of the module, making it useless.

Reply:
done in latest commit